diff --git a/node_modules/datatables.net-colreorder/types/types.d.ts b/node_modules/datatables.net-colreorder/types/types.d.ts
index e5dc283..1930c2b 100644
--- a/node_modules/datatables.net-colreorder/types/types.d.ts
+++ b/node_modules/datatables.net-colreorder/types/types.d.ts
@@ -7,7 +7,7 @@
 
 /// <reference types="jquery" />
 
-import DataTables, {Api} from 'datatables.net';
+import DataTables, { Api } from 'datatables.net';
 
 export default DataTables;
 
@@ -40,6 +40,8 @@ declare module 'datatables.net' {
 			/**
 			 * Create a new ColReorder instance for the target DataTable
 			 */
+      // Ignore this error: error TS7013: Construct signature, which lacks return-type annotation, implicitly has an 'any' return type.
+      // @ts-ignore
 			new (dt: Api<any>, settings: boolean | ConfigColReorder);
 
 			/**